ddpui/ddpairbyte/airbyte_service.py

```
@@ -939,7 +943,12 @@ def get_job_info_without_logs(job_id: str) -> dict:
 
 
 def get_jobs_for_connection(
-    connection_id: str, limit: int = 1, offset: int = 0, job_types: list[str] = ["sync"]
+    connection_id: str,
+    limit: int = 1,
+    offset: int = 0,
+    job_types: list[str] = ["sync"],
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)